Sources/TDS/Message/Login7Message.swift

```diff
@@ -21,7 +21,7 @@ extension TDSMessages {
         var language: String
         var database: String
         var sspiData: String
-        var atchDBFile: String = "" // TODO: What is this?
+        var atchDBFile: String = "" // the filename for a database that is to be attached during the connection process
```

Comment:
I believe this is something related to https://docs.microsoft.com/en-us/sql/relational-databases/databases/attach-a-database?view=sql-server-ver15

Reply:
Seems like this filed(and other) has validation on `dotnet's`  version https://github.com/dotnet/SqlClient/blob/66edd5801f0a94dc172788b294e02fd1223f2dce/src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/TdsEnums.cs#L614

Reply:
Thanks for adding the clarification.